packages/web/pages/index.tsx

```diff
@@ -1,24 +1,10 @@
+import { LoadingState } from '@metafam/ds';
```

Comment:
is this page necessary now that we are adding the redirect in the config itself?